host_guest/Isaacs_clip/host_guest/README.md

```diff
@@ -1,9 +1,9 @@
 # Host
 
-- Provided is the Trimertrip host for this challenge in PDB, Tripos MOL2, and SDF file format (trimertrip.mol2, trimertrip.sdf, trimertrip.pdb).
+- Provided is the Trimertrip host for this challenge in PDB, Tripos MOL2, and SDF file format (trimertrip.mol2, trimertrip.sdf, trimertrip.pdb). The structure was provided by Lyle Isaacs, from a crystal structure. The crystal structure will not be released. The original starting structure will be added here once the challenge ends
```

Comment:
Needs clarification as to "the structure". Specifically, I'd say something like "The structure was originally provided by Lyle Isaacs as a CIF file from crystallography. However, to avoid biasing results we are not releasing this structure at present, and instead equilibrated a structure of the host (`[detailed elsewhere in this repo](link)`) which we are providing here in sdf, mol2 and PDB formats. The original structure from crystallography will eventually be provided here once the challenge concludes."

Reply:
Updated!